pages/1.11/release-notes/1.11.0-rc4/index.md

```diff
@@ -56,7 +56,7 @@ Please try out the new features and updated data services. Provide any feedback
 [/enterprise]
 - IPv6 is now supported for Docker containers.
 - Performance improvements to the DC/OS network stack. All networking components (minuteman, navstar, spartan) are aggregated into a single systemD unit called `dcos-net`. Please read the note on [networking software re-architecture](/1.11/networking/#a-note-on-software-re-architecture) to learn more about the re-factoring of the network stack.
-- Configuration parameter `dns_forward_zones` now takes a list of objects instead of nested lists([DCOS_OSS-1733](https://jira.mesosphere.com/browse/DCOS_OSS-1733)). Please read the documentation for [dns_forward_zones](/1.11/installing/oss/custom/configuration/configuration-parameters/#dns-forward-zones) to understand its usage. 
+- Configuration parameter `dns_forward_zones` now takes a list of objects instead of nested lists ([DCOS_OSS-1733](https://jira.mesosphere.com/browse/DCOS_OSS-1733)). Please read the documentation for [dns_forward_zones](/1.11/installing/oss/custom/configuration/configuration-parameters/#dns-forward-zones) to understand its usage. 
 
 
 [enterprise]
```

Comment:
We are missing here:
https://jira.mesosphere.com/browse/DCOS-19050
https://jira.mesosphere.com/browse/DCOS-16510

CC: @mhrabovcin

Reply:
@vespian @mhrabovcin - Can you please provide a meaningful one-line description for the two issues above? I can't do that by looking at the JIRAs.